endpointmanager/pkg/archivefile/archivefile.go

```diff
@@ -312,14 +341,15 @@ func getHistory(ctx context.Context, args *map[string]interface{}) error {
 	}
 
 	// Get all rows in the history table between given dates
-	historyQuery := `SELECT url, updated_at, operation, capability_fhir_version, tls_version, mime_types FROM fhir_endpoints_info_history
-		WHERE updated_at between '` + ha.dateStart + `' AND '` + ha.dateEnd + `' AND url=$1 ORDER BY updated_at`
-	historyRows, err := ha.store.DB.QueryContext(ctx, historyQuery, ha.fhirURL)
+	historyQuery := `SELECT url, updated_at, operation, capability_fhir_version, tls_version, mime_types, requested_fhir_version FROM fhir_endpoints_info_history
```

Comment:
I've realized that here and in the `getMetadata` function, we don't really need to `SELECT` the url and requested_fhir_version, since we already have those!